selfservice/strategy/saml/strategy/strategy_auth.go

```diff
@@ -0,0 +1,61 @@
+package strategy
+
+import (
+	"errors"
+	"net/http"
+
+	"github.com/ory/kratos/identity"
+	"github.com/ory/kratos/selfservice/flow"
+	samlsp "github.com/ory/kratos/selfservice/strategy/saml"
+	"github.com/ory/x/sqlcon"
+)
+
+//#################
+// This file is called when the /ACS receives an assertion, the method below allows you to define whether to register or login the user indicated in the assertion
+//#################
+
+// Handle SAML Assertion and process to either login or register
```

Comment:
golang standard comment format starts with variable of function name.

Kratos uses a different one: https://github.com/ovh/kratos/blob/af1df63c62a8aec52f6adf9f23bb5dbd38c491ed/selfservice/flow/recovery/handler.go#L203

To be adapted based on your knowledge of ory coding style.